etc/containerpilot.json

```diff
@@ -20,11 +20,19 @@
       "interfaces": ["eth1", "eth0"]
     }{{ if .ACME_DOMAIN }},
     {
-      "name": "nginx-public-ssl",
+      "name": "nginx-ssl",
       "port": 443,
       "health": "/usr/local/bin/acme init && /usr/bin/curl --insecure --fail --silent --show-error --output /dev/null --header \"HOST: {{ .ACME_DOMAIN }}\" https://localhost/nginx-health",
       "poll": 10,
       "ttl": 25,
+      "interfaces": ["eth0"]
+    },
+    {
+      "name": "nginx-public-ssl",
```

Comment:
I'm not sure I understand this bit: why do we have two health checks of the same port?